gama.core/src/gama/core/kernel/batch/exploration/stochanalysis/StochanalysisExploration.java

```diff
@@ -186,7 +191,7 @@ public void explore(final IScope scope) throws GamaRuntimeException {
 				hasFacet(IKeyword.BATCH_OUTPUT)) { saveRawResults(scope, res_outputs); }
 		
 		/** If any of the two facet is missing pop up a warning */
-		if (hasFacet(IKeyword.BATCH_VAR_OUTPUTS) ||
+		if (hasFacet(IKeyword.BATCH_VAR_OUTPUTS) &&
```

Comment:
Shouldn't it be the opposite: ` ! (hasFacet(...) && hasFacet(...) )` ?
With the current code you will print a warning in the case both facets are present